### PR TITLE
Lvdb edit

### PR DESCRIPTION
**Pull request by lvdbrink**

### CODE REVIEW

File: assets/img/sdw-write.svg [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	.st0{fill:#FC4F08;}
+	.st1{fill:none;stroke:#FC4F08;stroke-width:6;stroke-miterlimit:10;}
+	.st2{fill-rule:evenodd;clip-rule:evenodd;fill:#FC4F08;}
+	.st3{fill-rule:evenodd;clip-rule:evenodd;fill:#FDC9B6;}
+	.st4{fill:#FDC9B6;stroke:#FC4F08;stroke-width:3.5;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;}
+	.st5{fill:#FFFFFF;}
+	.st6{fill:#60B530;}
+	.st7{fill:none;stroke:#60B530;stroke-width:5.1121;stroke-miterlimit:10;}
+	.st8{fill:none;}
+	.st9{font-family:'MyriadPro-Regular';}
```
Comment (tidoust) [inline assets/img/sdw-write.svg]: The `MyriadPro-Regular` font is not installed by default in browsers. Browsers will use a default font instead. In my case, that's `Times New Roman` which is far from ideal to render `{ ; }` and which breaks the alignment. I would suggest switching to `font-family: Sans-Serif;`, which should yield a good result across platforms.

Note the text also needs to be moved a bit, see below :)

File: assets/img/sdw-write.svg [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		<polygon class="st5" points="125,73 125,55.2 142.8,73 		"/>
+		<path class="st6" d="M128,62.5l7.5,7.5H128V62.5 M122,48v28h28L122,48L122,48z"/>
+	</g>
+	<g>
+		<path class="st6" d="M57,112.7c0,10.1-18.3,37.3-18.3,37.3s-18.3-27.2-18.3-37.3s8.2-18.3,18.3-18.3S57,102.6,57,112.7z"/>
+		<circle class="st5" cx="38.7" cy="112.7" r="8.3"/>
+	</g>
+</g>
+<rect x="9.3" y="12.9" class="st8" width="84.4" height="45.2"/>
+<text transform="matrix(1 0 0 1 17.3299 53.321)" class="st6 st9 st10">{ ; }</text>
```
Comment (tidoust) [inline assets/img/sdw-write.svg]: I suppose the text would appear centered in the box when the right font is used. If you switch to `Sans-Serif`, the matrix needs to be updated as well: `matrix(1 0 0 1 9 51)` seems good enough.

File: assets/img/sdw-write.svg (hunk 1 above)
Comment (lvdbrink) [reply assets/img/sdw-write.svg]: The artist made me a new version using a vector for the text instead of a font, is that okay too?

Comment (tidoust) [reply assets/img/sdw-write.svg]: Sure!

Comment (lvdbrink) [reply assets/img/sdw-write.svg]: I added the new version, can you recheck?

**Pull request merged**